scripts/rshim-fix.te

```diff
@@ -0,0 +1,32 @@
+module rshim-fix 1.3;
```

Comment:
Does it allow comment in this file to explain what it is?

Reply:
Comment added with new forced push.